scripts/services/search/search-gov.js

```diff
@@ -35,24 +34,34 @@ const translateFromSearchGov = (query) => (response) => {
         title: result.title,
         description: result.snippet
       };
-    }) : []
+    }) : [],
+    resultsCount: response.web ? response.web.total : null,
+    nextOffset: response.web ? response.web.next_offset : null,
+    total: response.total
   }
 };
 
-export const doSearchGovSearch = (query, highlightSearchTerms) => new Promise((resolve, reject) => {
+export const doSearchGovSearch = ({
+  query,
+  offset,
+  highlightSearchTerms,
+  searchTimeoutSeconds
+}) => new Promise((resolve, reject) => {
   const searchEndpoint = new URL(`${SEARCHGOV_ENDPOINT}/api/v2/search/i14y`);
-  const searchTimeout = 3; // seconds
 
-  window.setTimeout(() => {
-    reject(new Error(`Request for search.gov results timed out after ${searchTimeout} seconds.`));
-  }, searchTimeout * 1000);
+  if (searchTimeoutSeconds) {
+    window.setTimeout(() => {
```

Comment:
Does this setTimeout need to be cleared before you resolve?

Reply:
Since a promise can only resolve once, the reject wouldn't have any effect... maybe there's an argument we should clean it up anyway?

Reply:
No, I think that's fine.  I assumed that to be the case.